eng.ini

```diff
@@ -123,6 +124,7 @@ death.attack.onFire={%0} burned to death
 death.attack.outOfWorld={%0} fell out of the world
 death.attack.player.item={%0} was slain by {%1} using {%2}
 death.attack.player={%0} was slain by {%1}
+death.attack.thrownTrident={%0} was impaled to death by {%1}
```

Comment:
There is no such string with this key. Perhaps you meant [`death.attack.trident`](https://github.com/Mojang/bedrock-samples/blob/v1.19.40.2/resource_pack/texts/en_US.lang#L2394)?

Reply:
done.